signer/src/config/mod.rs

```diff
@@ -428,17 +434,10 @@ impl Validatable for SignerConfig {
 }
 
 impl SignerConfig {
-    /// Return the bootstrapped signing set from the config. This function
-    /// makes sure that the signing set includes the current signer.
+    /// Return the bootstrapped signing set from the config.
+    /// This function just convert [`Vec`] to [`BTreeSet`]
     pub fn bootstrap_signing_set(&self) -> BTreeSet<PublicKey> {
```

Comment:
We can deserialize to a `BTreeSet` if we want to. Then we could remove this function entirely. The only thing about that comes to mind about such a change is that we wouldn't be able to warn the operator if there were duplicates, but we don't do that now anyway.

Reply:
Idk.. For me is fine to have both `Vec` from field and `BTreeSet` from function.

Reply:
I'd be for deserializing directly to `BTreeSet`. We really don't care if there are duplicates, i.e. it's not an "error" per-se that requires preventing startup.

Reply:
Yeah, I tried to imagine why we may want to have `Vec` but failed. Here is migration to just btreeset: [27ab903](https://github.com/stacks-sbtc/sbtc/pull/1618/commits/27ab90374f0d308030bcde3b3dc2dfa62ea22ed3)